### DISCUSSION TITLE
在动态页面，b站似乎使用了新的话题框，动态过滤功能中原有的屏蔽话题框功能失效

### DISCUSSION THREAD

**Discussion by Kki426** [the1812/Bilibili-Evolved#2480] [提问 / Q&A] [answered]
如图，目前已无法屏蔽该话题框
![image](https://user-images.githubusercontent.com/90371201/140262764-8881c9da-2e44-4fd4-b0f8-cd9bdeee5daa.png)

**Comment by Geeyun-JY3** [comment, marked as answer]
重复：#2479

**Reply by Kki426** [reply]
好的，谢谢
